yargs.js

```diff
@@ -37,6 +37,11 @@ function Yargs (processArgs, cwd, parentRequire) {
 
   if (!cwd) cwd = process.cwd()
 
+  self.scriptName = function scriptName ($0) {
```

Comment:
No reason we can't use an intelligible parameter name here

Reply:
Saw it being used throughout (like [#L899](../../blob/master/yargs.js#L899)) so just went with it, but yeah I agree, shall change it to `scriptName`